samples/nrf9160/modem_shell/sample.yaml

```diff
@@ -5,70 +5,70 @@ tests:
     build_only: true
     integration_platforms:
       - nrf9160dk_nrf9160_ns
-    platform_allow: nrf9160dk_nrf9160_ns
+    platform_allow: nrf9160dk_nrf9160_ns nrf9161dk_nrf9161_ns
```

Comment:
I'm asking because I don't know and we should have a conscious decision about these:

1. Do we need to add `nrf9161dk_nrf9161_ns` also to `integration_platforms`?
2. What about the rest of the samples under nrf9160? So we need similar compilation for all of our samples for nrf9161dk_nrf9161_ns?

Reply:
1. AFAIK, we currently do not have any test setup in CI for nrf9161 DK. The plan is to add it to integration_platforms once we have test nodes with it. 
2. This is the samples and applications that it has been decided to support for now. More samples are to be supported later.

Reply:
Ok